message_definitions/v1.0/ras_a.xml

```diff
@@ -187,6 +187,18 @@
         <param index="4" label="Subject Follow Distance" units="m">Distance in 3D meters to attempt to maintain from target subject</param>
         <param index="5" label="Subject Follow Bearing" units="deg">Bearing in degrees from POI to attempt to maintain</param>
       </entry>
+      <entry value="51050" name="MAV_CMD_COMPONENT_CONTROL" hasLocation="false" isDestination="false">
+        <wip/>
+        <!-- This command is work-in-progress and it can therefore change. It should NOT be used in stable production environments. -->
+        <description>Control the state or functionality of system components.</description>
+        <param index="1" label="Component ID" enum="MAV_COMPONENT">Component ID of the system component to be controlled.</param>
+        <param index="2" label="Component Control Type" enum="COMPONENT_CONTROL">Control command type being sent to the system component.</param>
+        <param index="3" label="Component Instance ID" minValue="0" maxValue="1" increment="1">Component instance, when there is more than one instance per component.</param>
```

Comment:
@TSC21 Maybe I'm reading this wrong, but are you expecting that we only need 2 possible values for the Component Instance ID?

Reply:
`<param index="3" label="Component Instance ID" minValue="0" maxValue="100" increment="1">Identifier for the component instance when there is more than one instance of the component.  If there is only 1 instance of the component, then the sender should use value 0 and the receiver should ignore this field.</param>`

Reply:
Yep this is actually wrong. Good catch @radarku! Will update

Reply:
@radarku I made it a little bit more descriptive. Can you have a look?

Reply:
Looks great @TSC21 !  Nice additions - approved.